content/guide/agents-cloudify-agent.md

```diff
@@ -0,0 +1,188 @@
+---
+
+title: Agent Configuration
```

Comment:
Yes, I think I would expect to find all the information I need about Cloudify agents in one page.
What do you think?